pkger/clone_resource.go

```diff
@@ -155,6 +167,14 @@ func convertChartToResource(ch chart) Resource {
 		r[fieldChartBinSize] = ch.BinSize
 	}
 
+	if ch.BinCount != 0 {
+		r[fieldChartBinCount] = ch.BinCount
+	}
+
+	if ch.Position != "" {
+		r[fieldChartPosition] = ch.Position
```

Comment:
you can add this to the slice below that starts on line 188, ig ignores empty strings and doesn't provide the field 👍